examples/SimpleTxtPlugin/SimpleTxtPlugin.csproj

```diff
@@ -5,12 +5,12 @@
     <Nullable>enable</Nullable>
     <EnableDynamicLoading>true</EnableDynamicLoading>
     <GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>
-    <RootNamespace>Zeiss.FirstImportFormat</RootNamespace>
+    <RootNamespace>SimpleTxtPlugin</RootNamespace>
     <Authors>Carl Zeiss IMT GmbH</Authors>
     <Company>Carl Zeiss IMT GmbH</Company>
     <Product>Zeiss.FirstImportFormat</Product>
```

Comment:
Does it make sense to leave this property with the `Zeiss.` prefix?
I think this property only affects a single detail info value of the resulting DLL.

Reply:
Let's remove this completely.